requirements.in

```diff
@@ -19,7 +19,7 @@ boto3
 #-----------------------------------
 #   Excel Ingestion and Output
 #-----------------------------------
-pandas==1.5.*
+pandas==1.5.3
```

Comment:
I don't think this needed to change, but doesn't really _matter_ I guess since nothing else is going to be released in 1.5.x

Reply:
We should put in a ticket to upgrade pandas version.